tests/FakeItEasy.Tests.Approval/ApiApproval.ApproveApi40.approved.txt

```diff
@@ -597,12 +579,15 @@ namespace FakeItEasy.Core
         void OnAfterCallIntercepted(FakeItEasy.Core.ICompletedFakeObjectCall interceptedCall);
         void OnBeforeCallIntercepted(FakeItEasy.Core.IFakeObjectCall interceptedCall);
     }
-    public class Raise<TEventArgs> : FakeItEasy.Core.IEventRaiserArgumentProvider
-        where TEventArgs : System.EventArgs { }
+    public class Raise<TEventArgs>
+        where TEventArgs : System.EventArgs
+    {
+        public static System.EventHandler<TEventArgs> op_Implicit(FakeItEasy.Core.Raise<TEventArgs> raiser) { }
+        public static System.EventHandler op_Implicit(FakeItEasy.Core.Raise<TEventArgs> raiser) { }

```

Comment:
Also something I didn't notice before. Operators were not included. Another fix, nice!